packages/front-end/pages/dashboard.tsx

```diff
@@ -34,7 +35,7 @@ export default function Analysis(): React.ReactElement {
     <>
       <Head>
         <title>GrowthBook</title>
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" href={`${getPublicAssetsPath()}/favicon.ico`} />
```

Comment:
Pushing this to every url that needs the treatment feels like it would create a heavy maintenance load and also an easy footgun with public assets that are not prefixed appropriately.

I'd suggest to encapsulate this into a callback to make it clear what is the boundary and abstract away the actual implementation. Something like:
```typescript
const assetUrl: (path: string) => `${getPublicAssetsPath()}${path}`
```

And, then:
```suggestion
        <link rel="icon" href={assertUrl("/favicon.ico")} />
```

Or perhaps even an explicit module with all public asset urls exported?